packages/generate/src/queries.ts

```diff
@@ -313,3 +311,31 @@ export function ${functionName}(client: Executor${
       ];
   }
 }
+
+export const cardinalityToExecutorMethod = {
+  One: "queryRequiredSingle",
+  AtMostOne: "querySingle",
+  Many: "query",
+  AtLeastOne: "query",
+  Empty: "query",
+} satisfies Record<`${$.Cardinality}`, keyof Executor>;
+
+export class ImportMap extends Map<string, Set<string>> {
```

Comment:
I don't love that this is duplicated. It seemed like the easiest way though to handle an old driver version.